src/components/rootcomponents/productCardSkeleton.tsx

```diff
@@ -0,0 +1,11 @@
+export const ProductCardSkeleton = () => {
+  return (
+    <div>
+      <div className="grid grid-cols-3 gap-4">
+        <div className="h-48 bg-neutral-200 mb-4"></div>
+        <div className="h-48 bg-neutral-200 mb-4"></div>
+        <div className="h-48 bg-neutral-200 mb-4"></div>
```

Comment:
* Firstly, reimagine this and stop repeating yourself three times. Find a way to achieve the same functionality without making three separate divs
* Secondly you can make this a self closing tag since there's no data you're passing inside.